test/mason/masonInteractive.stdin

```diff
@@ -0,0 +1,4 @@
+project
```

Comment:
Ah, _stdin_, not  _in_. Sorry to lead you down the wrong path! Glad you were able to figure it out though :)

Reply:
No problem. 👍